frontend/app/services/redis-service.server.ts

```diff
@@ -28,14 +28,18 @@ function createRedisService() {
   // singleton redis client instance; lazy initialized as needed
   let redisClient: ReturnType<typeof createClient> | undefined = undefined;
 
-  async function getRedisClient() {
+  async function createRedisClient() {
     log.info(`Creating new Redis client; url=[${REDIS_URL}]`);
-    return (redisClient ??= await createClient({ url: REDIS_URL, username: REDIS_USERNAME, password: REDIS_PASSWORD })
+    return await createClient({ url: REDIS_URL, username: REDIS_USERNAME, password: REDIS_PASSWORD })
```

Comment:
No need to `await` if your wrapping function is `async`.. just return the promise.

Reply:
Noted - thanks

Reply:
And fixed